Remotion/Web/UnitTests/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRendererTest.cs

```diff
@@ -203,15 +203,14 @@ public void RenderWebStrings ()
 
       var titleNode = node.GetAssertedElementByID("DropDownMenu1_DropDownMenuLabel");
       Assert.That(titleNode.InnerXml, Is.EqualTo("Multiline<br />Title"));
-      var select = node.GetAssertedElementByClass("DropDownMenuSelect");
-      select.AssertAttributeValueEquals("title", "Multiline\nTitle");
     }
 
     private void AssertTitleSpan (XmlNode containerDiv, bool withTitle, bool withIcon)
     {
       var hasTitle = withTitle || withIcon;
       var titleDiv = containerDiv.GetAssertedChildElement("span", 0);
       titleDiv.AssertAttributeValueEquals("class", "DropDownMenuSelect");
+      titleDiv.AssertNoAttribute("title");
```

Comment:
```suggestion
```
I usually don't assert that an attribute is not there unless it's important for the behavior. In this case, it's just redundant information that we have removed. We don't need a regression-assertion for this.